src/main/java/com/gmail/yuramitryahin/dao/CinemaHallDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.gmail.yuramitryahin.dao;
+
+import com.gmail.yuramitryahin.exception.DataProcessingException;
+import com.gmail.yuramitryahin.lib.Dao;
+import com.gmail.yuramitryahin.model.CinemaHall;
+import com.gmail.yuramitryahin.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class CinemaHallDaoImpl implements CinemaHallDao {
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.persist(cinemaHall);
+            transaction.commit();
+            return cinemaHall;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert cinema hall " + cinemaHall, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<CinemaHall> getAll() {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<CinemaHall> getAllCommentsQuery = session
+                    .createQuery("from cinema_hall", CinemaHall.class);
```

Comment:
let's use class name instead of table name here

Reply:
fixed